app/controllers/api/jobs_controller.rb

```diff
@@ -8,7 +8,7 @@ def index
       JWT.decode bearer_token, hmac_secret, true, { algorithm: 'HS256' }
 
       jobs = Job.all
-      render status: 200, json: { data: jobs.as_json }
+      render status: 200, json: { data: jobs.order(sponsorship_level: :desc).as_json }
```

Comment:
This orders the jobs by sponsorship level in descending order when they are returned to the frontend (e.g. highest sponsorship levels come first).